zulipterminal/model.py

```diff
@@ -174,7 +174,7 @@ def __init__(self, controller: Any) -> None:
 
         self.users = self.get_all_users()
 
-        self.stream_dict: Dict[int, Any] = {}
+        self.stream_dict: Dict[int, Subscription] = {}
```

Comment:
This change, and possibly others in this commit seem much more related to those earlier in the PR than here at the end?